src/App.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+import './App.css';
+import { Route, Routes, Navigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Reservations from './components/pages/Reservations';
+import Login from './components/auth/Login'
+import Register from './components/auth/Register'
+import Splash from './components/auth/Splash'
+import Nav from './components/left/Nav';
+import Vehicles from './components/pages/Vehicles';
+import VehiclesDetails from './components/pages/VehiclesDetails';
+import AddVehicles from './components/pages/AddVehicles';
+import AddReservation from './components/pages/AddReservation';
+
+function App() {

```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.